exalearn-ui/src/app/pages/check-test-page/check-test-page.component.ts

```diff
@@ -8,7 +11,19 @@ import { EnglishLevels } from '../../enums/enums';
 	styleUrls: ['./check-test-page.component.scss']
 })
 export class CheckTestPageComponent {
-	data = MY_ASSIGNED_DATA;
-	isDataAvailable = true;
+	isDataAvailable = false;
 	tableColumns = ['level', 'expire', 'check'];
+	data: CheckCoachTest;
+
+	constructor(private user: UserService, private apiService: ApiService) {}
+
+	ngOnInit() {
+		this.apiService.getRequest(API_PATH.GET_UNCHECKED_TEST).then((data) => {
+			data.forEach((element: any) => {
+				element.levelType = Object.keys(EnglishLevels)[element.levelType - 1];
```

Comment:
Maybe you should name it level, not leveltype, so we didn't have same values with different names